coalib/settings/ConfigurationGathering.py

```diff
@@ -262,7 +262,9 @@ def get_config_directory(section):
 
     However if its origin is already a directory this will be preserved:
 
-    >>> section['files'].origin = os.path.abspath('/tmp/dir/')
+    >>> files = section['files']
+    >>> files.origin = os.path.abspath('/tmp/dir/')
+    >>> section.append(files)
```

Comment:
how's this related to this commit? (https://github.com/coala/coala/pull/3178/commits/2df2936b7f6448c568b6252304c1c6e90d65742e)